torch_patches/.torch_pin

```diff
@@ -0,0 +1 @@
+#48570
```

Comment:
If upstream pr already merged, can you remove the pin and rerun all of the CI?

Reply:
Oh right! I could remove `torch_pin` now.